index.html

```diff
@@ -12310,8 +12310,51 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<div class="property" id="aria-orientation">
 			<pdef>aria-orientation</pdef>
 			<div class="property-description">
-				<p><a>Indicates</a> whether the element's orientation is horizontal, vertical, or unknown/ambiguous.</p>
-				<p class="note">In ARIA 1.1, the default value for <pref>aria-orientation</pref> changed from <code>horizontal</code> to <code>undefined</code>. Implicit defaults are defined on some roles (e.g., <rref>slider</rref> defaults to horizontal; <rref>scrollbar</rref> defaults to vertical) but remain undefined on roles where an expected default orientation is ambiguous (e.g., <rref>radiogroup</rref>).</p>
+				<p>The <pref>aria-orientation</pref> attribute <a>indicates</a> whether the element's orientation is horizontal, vertical, or role-dependent/unknown/ambiguous.</p>
+
+			    <p>The <pref>aria-orientation</pref> attribute is an <a>enumerated attribute</a> with the following keywords and states:
+				<table>
+					<thead>
+						<tr>
+							<th scope="col">Keyword</th>
+							<th scope="col">State
+							<th scope="col">Description</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr>
+							<th class="value-name" scope="row"><code>horizontal</code></th>
+							<td>horizontal</td>
+							<td class="value-description">The element is oriented horizontally.</td>
+						</tr>
+						<tr>
+							<th class="value-name" scope="row"><code>undefined</code></th>
+							<td>undefined</td>
+							<td class="value-description">The element's orientation is unknown/ambiguous.</td>
+						</tr>
+						<tr>
+							<th class="value-name" scope="row"><code>vertical</code></th>
+							<td>vertical</td>
+							<td class="value-description">The element is oriented vertically.</td>
+						</tr>
+					</tbody>
+				</table>
+
+			    <p>The <pref>aria-orientation</pref> attribute's <a>invalid value default</a> and <a>missing value default</a> are the undefined state.
+
+				<p class="note">In ARIA 1.1, the default value for <pref>aria-orientation</pref> changed from <code>horizontal</code> to <code>undefined</code>.</p>
```

Comment:
These statements are seemingly in conflict unless we clarify the two different value concepts like "IDL value" and "accessibility internal value" (not particular to this name) ....

Reply:
Right, I was thinking we'd have a general distinction between "attribute" and "property" where the property is what AT computes the value of based on the "attribute". And I guess this note was about the "property" so should be moved a bit and clarified.

Reply:
I'm having trouble wrapping my head around this a bit, even though I do understand the problem you are trying to solve, IDL value/defaults vs "accessibility internal value"/defaults.

maybe we should use "implicit value" for the "Accessibility internal values"/defaults, as that is already what is in the role tables?

Maybe we should introduce some terms definitions in the PR so that it will be easier to understand where we are moving?

Reply:
I was using that to encompass things that the browser specifically does for AT when AT is enabled, which is somewhat separate implementation-wise and also in terms of what we're comfortable exposing. Whereas reflecting is more general and will also function if AT isn't enabled.

Reply:
Yes, what we'd be talking about is not the default value of the DOM property, where el.ariaOrientation would always return `undefined`, but something like a new webdriver getter like `computedAriaOrientation`, or `computedAccessibleNode.orientation` where the default value on a range could be `horizontal`